docs/products/clickhouse/howto/use-play.rst

```diff
@@ -0,0 +1,14 @@
+Use ClickHouse® play UI
+========================
+
+ClickHouse® includes a built-in user interface for running SQL queries. You can access it through HTTPS protocol. Follow these steps to use it:
+
+1. Log in to the `Aiven web console <https://console.aiven.io/>`_ and select your ClickHouse service.
+#. In the *Overview* tab find *Connection information* and select **ClickHouse HTTPS**
```

Comment:
In the *Overview* tab, find *Connection information* and select **ClickHouse HTTPS**.

(a ',' and '.' added).